python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/test_translator.py

```diff
@@ -0,0 +1,30 @@
+import uuid
```

Comment:
nit; why is this in translator tests? Shouldn't it be in the same place as the previous workspace method tests?

Reply:
Done in [697e460](https://github.com/dagster-io/dagster/pull/25797/commits/697e460a645a17745f602ad0af25b077356cf177)